x/ccv/provider/keeper/keeper_test.go

```diff
@@ -197,3 +205,243 @@ func (suite *KeeperTestSuite) TestInitHeight() {
 		suite.Require().EqualValues(t.expected, height)
 	}
 }
+
+func (suite *KeeperTestSuite) TestHandleSlashPacketDistribution() {
+	providerStakingKeeper := suite.providerChain.App.GetStakingKeeper()
+	providerSlashingKeeper := suite.providerChain.App.(*appProvider.App).SlashingKeeper
+	ProviderKeeper := suite.providerChain.App.(*appProvider.App).ProviderKeeper
+
+	// bonded amount
+	bondAmt := sdk.NewInt(1000000)
```

Comment:
This is fine, but overall in the tests we should try to keep magic numbers a bit under control. For example I think this is the DefaultPowerReduction so better to use that. 

I made an issue as this is a widespread phenomena in the existing code. I'm not singling this PR, or you, out.